clients/android/app/src/main/java/app/lockbook/utils/LogUtil.kt

```diff
@@ -0,0 +1,21 @@
+package app.lockbook.utils
+
+import java.io.BufferedReader
+import java.io.InputStreamReader
+
+class LogUtil {
```

Comment:
What is this

Reply:
This is a class I created to get logs out of android if we need them. Right now it is unused, but if in future branches we wanted to get the logs we could use this.